src/excalidraw-app/index.tsx

```diff
@@ -297,6 +310,33 @@ function ExcalidrawWrapper(props: { collab: CollabAPI }) {
       }
     }
   };
+
+  const renderLanguageList = (isMobile: boolean) => (
+    <LanguageList
+      onChange={(lng) => {
+        setLang(lng);
+      }}
+      languages={languages}
+      floating={!isMobile}
+      currentLanguage={lang}
+    />
+  );
+  const renderFooter = (isMobile: boolean) => {
+    if (isMobile) {
+      return (
+        <fieldset>
+          <legend>{t("labels.language")}</legend>
+          {renderLanguageList(isMobile)}
+        </fieldset>
+      );
+    }
+    return renderLanguageList(isMobile);
+  };
```

Comment:
I'd like to make this into a component, but it seems we're stuck on `isMobile` not being a parent of this component, which we'll need to figure out in a different PR. So, for now, let's just memoize the `renderFooter` function via `useCallback`.

Reply:
isMobile can not become parent of this component anymore as its not part of child but since host(Excal app) is separate now we can wrap the mobile provider instead and then we need not pass `isMobile` from Excalidraw to host.
Wdyt ?

Reply:
That would potentially diverge in behavior since the host app's `isMobile` may not reflect Excalidraw's `isMobile` (if we e.g. later change the implem to check container size instead of `window` size — which we may want to).

That said, I have no clue how to go about this right now. Best leave that part for later.